tests/lwip/udp.c

```diff
@@ -121,7 +121,7 @@ static int udp_send(char *addr_str, char *data, unsigned int num,
             printf("Success: send %u byte over UDP to %s\n",
                    (unsigned)data_len, addr_str);
         }
-        xtimer_usleep(delay);
+        ztimer_sleep(ZTIMER_USEC, delay);
```

Comment:
something like 
d5dee30368b2d21d2a9c5f1bd632e375229ce2b1
is missing here  but this might be an oportunity for another PR

```suggestion
        if( i+1 < num) ztimer_sleep(ZTIMER_USEC, delay);
```

Reply:
True but IMO not worth it, this is just for the test, the actual module is only using `msec`